### ISSUE TITLE
Accents missing in Spanish infinitives

### ISSUE THREAD

**Issue by oowowaee** [clips/pattern#163]
Accented Spanish infinitives are missing their accents.  I have fixed this in my local branch - should I make a PR, or were these omitted intentionally?

**Comment by canarvaeza** [comment]
Oh men they should give you the opportunity to upgrade spanish, we could start a group to improve the pattern.es module.

**Comment by oowowaee** [comment]
@canarvaeza Yeah, the more I used the library the more issues I have found with it...there is a lot that could be done to improve the Spanish module.  It's far from perfect, or necessarily generalizable, but you can checkout my fork to see some of the issues I have come across.
